dub.json

```diff
@@ -21,17 +25,33 @@
         },
     },
 
+    "subPackages": [
+        {
+            "name": "configure",
+            "targetType": "executable",
```

Comment:
> Oh, I have missed that. I was rdmd script before, wasn't it? Why have you changed that?

To let Dub automatically handle identifying the compiler, i.e. support both DMD and LDC.

> Alternatively you can move the package definition to configure.d itself

I did that first but run into this problem: https://github.com/dlang/dub/issues/910. Then I though what's the point of having it defined there if I cannot use `./configure` anyway. But as you mention now, it's better to avoid exposing the sub package at all.